pom.xml

```diff
@@ -31,8 +31,15 @@
         <munit.input.directory>src/test/munit</munit.input.directory>
         <munit.citizen.input.directory>src/test/munit-citizen</munit.citizen.input.directory>
         <munit.output.directory>${basedir}/target/test-mule/munit</munit.output.directory>
-        <munit.extensions.maven.plugin.version>1.2.0-SNAPSHOT</munit.extensions.maven.plugin.version>
-        <munit.version>3.1.0-SNAPSHOT</munit.version>
+        <munit.extensions.maven.plugin.version>1.2.0-rc1</munit.extensions.maven.plugin.version>
+        <munit.version>3.1.0-rc2</munit.version>
```

Comment:
New versions are available for 
munit 3.1.0-rc3
munit-extensions-maven-plugin 1.2.0-rc2

Make sure you use those.